editor/sidebar/header.js

```diff
@@ -48,9 +49,16 @@ export default connect(
 	( state ) => ( {
 		panel: getActivePanel( state ),
 		count: getSelectedBlockCount( state ),
+		isMobile: ! state.responsive.greaterThan.medium,
```

Comment:
We have 3 places where we need to pull `isMobile` value. Every time this value changes, React will detect that props have changed so it will try to re-render component even though `onToggleSidebar` method is not dynamically created. I'm wondering if this component will re-render when `isMobile` value changes with all those optimizations applied. It's a very interesting use case :)